website/content/accessibility/deaf-and-hard-of-hearing/conference-best-practices.md

```diff
@@ -103,11 +111,9 @@ If you only have a few deaf individuals attending, provide two interpreters per
 
 Of course, this approach doesn’t scale well. If you have more deaf attendees, you’ll need to share accommodations among more individuals.
 
-Before the conference, ask attendees to sign up for sessions in advance so you know which sessions must be staffed by interpreters. But allow for last-minute changes and be prepared to inform interpreters about these. While you’ll want to plan ahead, you also want to offer attendees the flexibility to change their minds. Maybe they met someone who’s giving a talk, and they’d like to attend that session. If you have multiple deaf attendees, have interpreters at keynotes by default.
```

Comment:
I'd skip this. It adds more complexity, while we recommend focusing on captions for the technical sessions anyway. Unfortunately, interpreters won't be able to interpret them accurately.

Reply:
Sure Catherine. Fine with me.